scala2.13/aggregator/pom.xml

```diff
@@ -0,0 +1,591 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2021-2023, NVIDIA CORPORATION.
```

Comment:
shouldn't have 2021 in copyright

Reply:
So this, I don't know an easy way to fix this because the files are generated.  At some point I also wanted to add a header explaining the file was generated as well.  So I would have to update the workflow that generates these files, and manage those copyrights into the future. I'm open to suggestions on this.

Reply:
oh right, lets leave it